assets/js/tracking.js

```diff
@@ -19,15 +19,15 @@ $(document).ready(function() {
             // Create the tracking object.
             const trackingData = {
                 // The id of the element.
-                element_id: $(element).attr("id"),
+                element_id: $(element).attr("data-tracking-id"),
```

Comment:
A bit paranoid :), but I'll add some code to handle if the element doesn't exist. The way the script works is that only links found on the page will be fed into the function. But this is the world of browser javascript so it's probably best to err on the side of caution.

If the attribute isn't present it won't cause a "problem" for the user. We will just have useless data to look at. The id should always be present though because the I query for links with `data-tracking-id` and then that list is looped over and fed into this function.